### ISSUE TITLE
ObjectTracking is returning null

### ISSUE THREAD

**Issue by ArtanBerisha1** [tensorflow/tensorflow#41554]
I implemented tensorflow/example/android.../tracking into my android app and when I am trying to activate and disable ObjectDetection, then I am getting an error : 
'CHECK FAILED (object_tracker != NULL): null object tracker!' 

In ObjectTracker.java there is a documentation like this:
**
 * True object detector/tracker class that tracks objects across consecutive preview frames.
 * It provides a simplified Java interface to the analogous native object defined by
 * jni/client_vision/tracking/object_tracker.*.
 *
 * Currently, the ObjectTracker is a singleton due to native code restrictions, and so must
 * be allocated by ObjectTracker.getInstance(). **In addition, release() should be called**
 * as soon as the ObjectTracker is no longer needed, **and before a new one is created.**
 *
 * nextFrame() should be called as new frames become available, preferably as often as possible.
 *
 * After allocation, new TrackedObjects may be instantiated via trackObject(). TrackedObjects
 * are associated with the ObjectTracker that created them, and are only valid while that
 * ObjectTracker still exists.
 */

I put a Log in release() method and I saw that this method gets called only once when ObjectTracker is no longer needed but not when a new one is created, 
where should I call the release() method before new one is created?

**Comment by ravikyram** [comment]
@ArtanBerisha1 

Request you to fill [issue template.](https://github.com/tensorflow/tensorflow/issues/new/choose).
Please provide details about what platform you are using (operating system, architecture).

Request you to share colab link or simple standalone code to reproduce the issue in our environment.It helps us in localizing the issue faster.Thanks!
